### ISSUE TITLE
The codes is not designed for RGB images

### ISSUE THREAD

**Issue by hermosayhl** [sjmoran/neural_curve_layers#2]
I have made lots of efforts to run the codes with MIT-Adobe FiveK datasets. However, the codes are designed for RAW images rather than RGB pairs. It's consuming my time.
Would you please release one version of codes for RGB images?

**Comment by sjmoran** [comment]
Hi hermosayhl, firstly thank you for your interest in our work! The RGB variant of the low-level network is similar to the RAW, but you are correct that there are some key differences (see supplementary material of the paper). I will upload an RGB network for you.

**Comment by hermosayhl** [comment]
> Hi hermosayhl, firstly thank you for your interest in our work! The RGB variant of the low-level network is similar to the RAW, but you are correct that there are some key differences (see supplementary material of the paper). I will upload an RGB network for you.

### Thanks for your reply!

I have managed to run the codes with RGB image pairs although some hyber parameters are changed. As long as one can dedicated himself to the codes.

I'm not sure whether my changed version is correct. However, the initial trained results are pleasing so far.
After I finished training, I may upload the changed codes and pretrained models for all friends.

**Comment by hermosayhl** [comment]
> Hi hermosayhl, firstly thank you for your interest in our work! The RGB variant of the low-level network is similar to the RAW, but you are correct that there are some key differences (see supplementary material of the paper). I will upload an RGB network for you.

Listed below are some of the images for testing:

![image](https://user-images.githubusercontent.com/49510831/92926126-26a2ba00-f46e-11ea-8c35-fdfca53543d6.png)

![image](https://user-images.githubusercontent.com/49510831/92926159-328e7c00-f46e-11ea-99d9-86eac0d692f2.png)

![image](https://user-images.githubusercontent.com/49510831/92926186-3cb07a80-f46e-11ea-91ca-00eb7ade44f4.png)

**Comment by sjmoran** [comment]
Good to hear that you got the RBG variant working. And thank you for contributing the code and trained models to the community. To clarify, in your output images, from left-to-right: the left most image is the input. Which of the other two images are the prediction and ground-truth?

**Comment by hermosayhl** [comment]
> Good to hear that you got the RBG variant working. And thank you for contributing the code and trained models to the community. To clarify, in your output images, from left-to-right: the left most image is the input. Which of the other two images are the prediction and ground-truth?

### Firstly
the images produced are: [input image,   label image,  enhanced image].
### Secondly
The training process is coming to the end. I will change the code and trained models as soon as possible.
